packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -888,12 +888,11 @@ function* transferStep({
     localStorageState
   )
 
-  // Update wallet balance optimistically
-  yield* put(
-    increaseBalance({
-      amount: audioTransferredWei.toString() as StringWei
-    })
-  )
+  // Refetch wallet balance
```

Comment:
def make sure to test this - worried this could refetch before the RPC has the balance change. Might make more sense to optimistically update